docs/stylesheets/style.css

```diff
@@ -632,11 +632,13 @@ p {
 @media only screen and (max-width: 768px) {
   .nav__items {
     flex-direction: column;
+    display: flex;
```

Comment:
this is probably not needed - will be inherited from line 617 if not overwritten by default!

Reply:
Thanks! Tested and removed this change